x-pack/plugins/reporting/server/export_types/common/export_type.ts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  IBasePath,
+  Headers,
+  Logger,
+  CoreKibanaRequest,
+  CoreSetup,
+  FakeRawRequest,
+  HttpServiceSetup,
+  KibanaRequest,
+  PluginInitializerContext,
+  SavedObjectsClientContract,
+  SavedObjectsServiceStart,
+  UiSettingsServiceStart,
+  IClusterClient,
+} from '@kbn/core/server';
+import { LicenseType } from '@kbn/licensing-plugin/common/types';
+import { ScreenshottingStart } from '@kbn/screenshotting-plugin/server';
+import { DEFAULT_SPACE_ID } from '@kbn/spaces-plugin/common';
+import { SpacesPluginSetup } from '@kbn/spaces-plugin/server';
+import { ReportingConfigType } from '../../config';
+import { ReportingServerInfo } from '../../core';
+import { CreateJobFn, ReportingStart, RunTaskFn } from '../../types';
+
+/**
+ * @TODO move to be within @kbn-reporting-export-types
+ */
```

Comment:
Let's leave this TODO out for now, since you've also been actively working on this :) 
```suggestion
```